apps/programmes/urls.py

```diff
@@ -1,11 +1,11 @@
 from django.urls import path, include
 from rest_framework.routers import DefaultRouter
-
-from apps.programmes.views import ProgrammeViewSet
+from apps.programmes.views import ProgrammeViewSet, ProgrammeStructureView
 
 router = DefaultRouter()
 router.register("programmes", ProgrammeViewSet, basename="programme")
 
 urlpatterns = [
     path("", include(router.urls)),
+    path("programmes/<int:programme_id>/structure", ProgrammeStructureView.as_view(), name="programme-structure"),
```

Comment:
programme_id is uuid not int

Reply:
shouldn't the url be `programmes/id/courses` tho? it seems that you'd like to get the courses of the programme

Reply:
Yeah I forgot about the UUID change and did that to int only. Correcting it